input/templates/partials/common/header.json

```diff
@@ -1,673 +1,6 @@
 {
 	"title" : "Sunrise - Page Title",
```

Comment:
Really? I usually see it like this. But it's a fake title anyway, let's put the effort here in the implementation side.